### ISSUE TITLE
NTR: breast fibroadenosis

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5096]
**Preferred term label:**
breast fibroadenosis

**Synonyms**
fibroadenosis - breast
fibroadenosis of breast

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2429655/?page=1

**Parent term (use hpo.jax.org/app)**
Breast mass: https://hpo.jax.org/app/browse/term/HP:0032408

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

See related ticket in Mondo:
https://github.com/monarch-initiative/mondo/issues/429

**Comment by pnrobinson** [comment]
I think the phenotypic observation should be called breast fibroadenoma, which maps to this NCI term
https://ncithesaurus.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&version=20.02d&ns=ncit&code=C3744&key=n27591869&b=1&n=null#

**Comment by pnrobinson** [comment]
I see that HPO already has a term for this:
https://hpo.jax.org/app/browse/term/HP:0010619
I will add the above synonyms and the link to the NCI term.
